modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v11_2_1/CommercePermissionUpgradeProcess.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.internal.upgrade.v11_2_1;
+
+import com.liferay.account.model.AccountEntry;
+import com.liferay.commerce.constants.CommerceAccountActionKeys;
+import com.liferay.portal.kernel.model.ResourceAction;
+import com.liferay.portal.kernel.model.ResourcePermission;
+import com.liferay.portal.kernel.security.permission.ActionKeys;
+import com.liferay.portal.kernel.service.ResourceActionLocalService;
+import com.liferay.portal.kernel.service.ResourcePermissionLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Stefano Motta
+ */
+public class CommercePermissionUpgradeProcess extends UpgradeProcess {
+
+	public CommercePermissionUpgradeProcess(
+		ResourceActionLocalService resourceActionLocalService,
+		ResourcePermissionLocalService resourcePermissionLocalService) {
+
+		_resourceActionLocalService = resourceActionLocalService;
+		_resourcePermissionLocalService = resourcePermissionLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		ResourceAction resourceAction =
+			_resourceActionLocalService.fetchResourceAction(
+				AccountEntry.class.getName(), ActionKeys.UPDATE);
+
+		if (resourceAction == null) {
+			return;
+		}
+
+		for (ResourcePermission resourcePermission :
+				_resourcePermissionLocalService.getResourcePermissions(
+					AccountEntry.class.getName())) {
+
+			if (_resourcePermissionLocalService.hasActionId(
+					resourcePermission, resourceAction) &&
+				(resourcePermission.getScope() != 4)) {
```

Comment:
I don't get this, what is 4?

Reply:
It's the same we did here (https://github.com/liferay/liferay-portal/blob/de0de3137f8e3a749bcbb6d62dc186a4e734393e/modules/apps/account/account-service/src/main/java/com/liferay/account/internal/upgrade/v2_10_1/AccountRoleResourceUpgradeProcess.java#L4). 

4 should be the "SCOPE_INDIVIDUAL"

Reply:
if you find a constant that's easy to use, please use the constant

Reply:
Done